docs/admin/pool_operations.md

```diff
@@ -916,6 +916,26 @@ and possibly repair a pmemobj file. As discussed in the previous section, the
 rebuild status can be consulted via the pool query and will be expanded
 with more information.
 
+## Pool-Wise Redundancy Factor (PW_RF)
+
+When there are cascading multiple failures, if the number of failed fault
+domains exceeds the PW_RF, there could be unrecoverable errors and applications
+could suffer from data loss. Especially when there is power or network outage,
+it could cause massive node/engine failures. In most cases those failures can
+be recovered and DAOS engines can be restarted and work again.
```

Comment:
If the DAOS system experiences cascading failures, where the number of failed fault domains exceeds a pool's redundancy factor, there could be unrecoverable errors and applications could suffer from data loss. This can happen in cases of power or network outages and would cause node/engine failures. In most cases those failures can be recovered and DAOS engines can be restarted and the system can function again.

Reply:
ok